crates/nargo/Cargo.toml

```diff
@@ -29,10 +29,11 @@ git-version = "0.3.5"
 hex = "0.4.2"
 termcolor = "1.1.2"
 tempdir = "0.3.7"
+sha2 = "0.10.6"
```

Comment:
We can remove this now following https://github.com/noir-lang/noir/pull/765#discussion_r1107794066